offload_vio/device_tx/plugin.cpp

```diff
@@ -1,91 +1,239 @@
 #include "common/plugin.hpp"
 
 #include "common/data_format.hpp"
+#include "common/network/net_config.hpp"
+#include "common/network/socket.hpp"
+#include "common/network/timestamp.hpp"
 #include "common/phonebook.hpp"
+#include "common/stoplight.hpp"
 #include "common/switchboard.hpp"
+#include "common/threadloop.hpp"
+#include "video_encoder.h"
 #include "vio_input.pb.h"
 
-#include <ecal/ecal.h>
-#include <ecal/msg/protobuf/publisher.h>
+#include <boost/lockfree/spsc_queue.hpp>
+#include <cstdlib>
+#include <ctime>
+#include <filesystem>
+#include <fstream>
+#include <opencv/cv.hpp>
 #include <opencv2/core/mat.hpp>
 
 using namespace ILLIXR;
+#define USE_COMPRESSION
```

Comment:
Should disable this if GStreamer is not installed by default with ILLIXR.
Alternatively, need to update the install script with GStreamer.

Reply:
Agreed.